.circleci/config.yml

```diff
@@ -76,6 +76,11 @@ jobs:
       - restore_cache:
           keys:
             - daml-{{ checksum "daml.yaml" }}
+      - run:
+          name: Install libnss3 for Chrome
+          command: |
+            sudo apt-get update || sudo apt-get update
+            sudo apt install libnss3
```

Comment:
Why do we need this? As far as I know Puppeteer will install a Chromium of its own. Does that not work properly? 🤔

Reply:
We need this, because Chromium won't start. (see failing CI jobs earlier, look for nss in git log).
I followed this: https://github.com/alixaxel/chrome-aws-lambda/issues/164#issuecomment-754621407